doc/develop/application/index.rst

```diff
@@ -191,25 +191,27 @@ following example, ``app`` is a Zephyr freestanding application:
             └── main.c
 
 Example workspace application
-******************************
+*****************************
 
-A reference :ref:`workspace application <zephyr-workspace-app>` contained in its
-own Git repository can be found in the `Example Application`_ repository.
-It can be used as a reference on how to structure out-of-tree, Zephyr-based workspace applications
-using the :ref:`T2 star topology <west-t2>`. It also demonstrates the out-of-tree
-use of features commonly used in applications such as:
+The `example-application`_ Git repository contains a reference :ref:`workspace
+application <zephyr-workspace-app>`. It can be used as a reference when
+creating your own application.
 
-- Custom boards
-- Custom devicetree bindings
-- Custom drivers
-- Continuous Integration (CI) setup
-- An example west :ref:`extension <west-extensions>` command
+The example-application repository demonstrates how to use several
+commonly-used features, such as:
+
+- Custom :ref:`board ports <board_porting_guide>`
+- Custom :ref:`devicetree bindings <dt-bindings>`
+- Custom :ref:`device drivers <device_model_api>`
+- Continuous Integration (CI) setup using :ref:`twister <twister_script>` via
```

Comment:
I think this is not entirely accurate. CI builds the application without using twister for example. Maybe add an extra point talking about tests written using ztest and run with twister.